jobs/config.json

```diff
@@ -2025,6 +2025,23 @@
       "UNKNOWN"
     ]
   },
+  "ci-kubernetes-e2e-gce-gci-qa-m63": {
+    "args": [
+      "--check-leaked-resources",
+      "--env-file=jobs/platform/gce.env",
+      "--extract=gci/gci-63",
+      "--gcp-project-type=gci-qa-project",
```

Comment:
I didn't check but please make sure gci-63 this exists before checking in.

Reply:
It does now. Thanks for reminder.